website/src/components/layout/layout.js

```diff
@@ -57,7 +57,7 @@ const Wrapper = (props) => {
 		return (
 			<GEL brand={brandOverrides(brands['WBC'])}>
 				<Global styles={gelFonts} />
-				<Normalize />
+				{/* <Normalize /> */}
```

Comment:
Can likely be deleted?

Reply:
I think so yes... not sure why they were there. We have the exact reset styling in Core already